### PR TITLE
Manage organisation page for system admins

### PR DESCRIPTION
**Pull request by alvak1**
Testing instructions

After login for system admins - they should be redirected to manage organisations page instead of dashboards page

### CODE REVIEW

**Review by alainajaffer — COMMENTED** [review]
When i am logged in as a system admin and it is letting me access create disruptions, /view-all-disruptions and /dashboard

**Comment by alvak1** [comment]
> When i am logged in as a system admin and it is letting me access create disruptions, /view-all-disruptions and /dashboard

done in the other PR from branch feat/Disruption-242_add_org

File: site/pages/sysadmin/users.page.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { ReactElement } from "react";
+
+const AdminUsers = (): ReactElement => {
+    return <>Placeholder</>;
```
[the remark this answers is not in the document]
Comment (alvak1) [reply site/pages/sysadmin/users.page.tsx:4]: Is updated in the other PR

**Comment by alvak1** [comment]
Closing as this PR is merged with another PR

**Pull request closed without merging**